src/lib/api/lists.ts

```diff
@@ -0,0 +1,13 @@
+import { config } from 'config'
+import { BaseAPI } from 'decentraland-dapps/dist/lib/api'
+
+export const DCL_LISTS_URL = config.get('DCL_LISTS_SERVER', '')
+
+export class DclLisstAPI extends BaseAPI {
```

Comment:
There is a typo here in `Lists`. Can you fix it 🙏 ?

Reply:
done! thanks :)